pandas/core/frame.py

```diff
@@ -1940,7 +1944,7 @@ def to_stata(
         >>> df.to_stata('animals.dta')  # doctest: +SKIP
         """
         kwargs = {}
-        if version not in (114, 117, 118):
+        if version not in (114, 117, 118, 119, None):
             raise ValueError("Only formats 114, 117 and 118 are supported.")
```

Comment:
This error message will need to be updated to include 119: `"Only formats 114, 117, 118, and 119 are supported."`

Reply:
Fixed, thanks.